src/components/MultiLink/Legend.vue

```diff
@@ -58,6 +59,45 @@ export default Vue.extend({
     nodeColorScale() {
       return store.getters.nodeColorScale;
     },
+
+    columnTypes() {
+      const typeMapping: { [key: string]: string} = {};
+
+      if (store.getters.networkMetadata !== null) {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        Object.entries(store.getters.networkMetadata).forEach(([tableName, metadata]) => {
+          (metadata as TableMetadata).table.columns.forEach((columnType) => {
+            typeMapping[columnType.key] = columnType.type;
+          });
```

Comment:
Totally optional suggestion but this could be better expressed as a call to `.map()`.

Reply:
Turns out a map call here won't work; my bad.